R/zchunk_L203.demand_input.R

```diff
@@ -37,9 +37,9 @@ module_aglu_L203.demand_input_DISABLED <- function(command, ...) {
              "L101.Pop_thous_R_Yh",
              "L102.pcgdp_thous90USD_Scen_R_Y"))
   } else if(command == driver.DECLARE_OUTPUTS) {
-    return(c("L203.SectorLogitTables[[ curr_table ]]$data",
+    return(c("L203.SectorLogitTables[[curr_table]]",
```

Comment:
There are still some funkiness about these outputs. It seems can't find the correct comparison data to test against.